src/slack_interaction_api_util.ts

```diff
@@ -1,11 +1,19 @@
 import axios from 'axios';
+import axiosRetry from 'axios-retry';
 import { voterStatusPanel, SlackBlock } from './slack_block_util';
 import logger from './logger';
 import { UserInfo } from './types';
 import * as SlackApiUtil from './slack_api_util';
 import * as SlackInteractionHandler from './slack_interaction_handler';
 import { PromisifiedRedisClient } from './redis_client';
 
+axiosRetry(axios, {
```

Comment:
Can we just import the `slackAPI` client from the other file and use that? Should also simplify all the axios calls in this file

Reply:
Yep! Did I do this correctly?:

1) Will the export include the modifications to slackAPI util in-file (e.g. retries)?
2) Did I handle params nesting for slackAPI methods correctly?
3) Did I preserve get/post methods correctly?